index.html

```diff
@@ -51,7 +51,70 @@ <h2 class="describe">I’am a Software Developer</h2>
         </ul>
         
     </section>
-    
+    <section class="works">
+        <div class="title">
+            <h1 class="worktext">My Recent Works</h1>
+        </div>
+        <div class="workframe">
+
+        </div>
+        <div class="wk">
+        <div class="project1">
+            <p class="post">Multi-Post Stories</p>
+            <p class="subpost">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever
+                 since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        
+        <ul class="worka">
+            <li class="langa">css</li>
+            <li class="langb">html</li>
+            <li class="langc">javascript</li>
+            <li class="langd">ruby</li>
+        </ul>
+        <button class="buttonw"><ins class="seep">See Project</ins></button>
```

Comment:
- [ ] Kindly make sure you stick to the design as much possible as you can. Please the project image here on the project section as it is in the figma template

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198567394-39d19af4-4a1c-4682-b138-4152120f0fce.png)
![image](https://user-images.githubusercontent.com/97100091/198567590-fef5ec80-61f7-46fa-abc8-a4d706f414ff.png)

![image](https://user-images.githubusercontent.com/97100091/198567634-242fe7c8-78a0-489c-b359-eb912d699095.png)
![image](https://user-images.githubusercontent.com/97100091/198567795-5c2da52c-cfbd-43d3-91da-ae5f8a6f5994.png)
![image](https://user-images.githubusercontent.com/97100091/198567842-31e06975-1615-4416-8477-41e7dd194cd4.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198567936-e30c348b-03bd-44be-9be3-d8c83159a1d7.png)

![image](https://user-images.githubusercontent.com/97100091/198567992-dfa3d96c-db0e-4190-a0fb-54484afb69f8.png)
![image](https://user-images.githubusercontent.com/97100091/198568063-3e2fa1cb-8731-4478-a1f4-6fb626dbc8ba.png)
![image](https://user-images.githubusercontent.com/97100091/198568156-f288216f-8953-4d23-83ef-df01a285a094.png)